web/src/index.html

```diff
@@ -14,19 +14,25 @@
 limitations under the License.
 -->
 
-<!doctype html>
```

Comment:
Is this needed in this PR?

Reply:
Prettier did it.. Should we allow Prettier to reformat as we go, or should we do a global reformat separately?

Reply:
We can keep this reformatting in this PR for simplicity, these comment add enough context. I think it's a good idea to run a general reformatting in a separate PR.